common/lib/xmodule/xmodule/html_module.py

```diff
@@ -145,7 +145,12 @@ def studio_view(self, _context):
         ],
         'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
     }
-    preview_view_css = {'scss': [resource_string(__name__, 'css/html/display.scss')]}
+    preview_view_css = {
+        'scss': [
+            resource_string(__name__, 'css/html/display.scss'),
+            resource_string(__name__, 'css/html/esme_custom.scss'),
+        ]
```

Comment:
Nit: Leave a trailing `,` to reduce dif noise when adding new entries in the future.